package.json

```diff
@@ -22,5 +22,6 @@
     "url": "git+https://github.com/measurement-factory/anubis.git"
   },
   "author": "The Measurement Factory",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "type": "module"
```

Comment:
Please move this new "type" property to position it between "description" and "main" entries above.